selendroid-common/src/main/java/io/selendroid/common/SelendroidCapabilities.java

```diff
@@ -395,17 +395,13 @@ private String getDefaultVersion(Set<String> keys, String appName) {
     return listOfApps.size() > 0 ? listOfApps.last() : null;
   }
 
-  // throws exception if user didn't pass the capability as a boolean
+  // throws exception if user didn't pass the capability as a boolean or String parsable as boolean
   private Boolean getBooleanCapability(String key) {
     Object o = getRawCapabilities().get(key);
     if (o == null) {
       return null;
-    } else if (o instanceof Boolean) {
-      return (Boolean) o;
     } else {
-      throw new ClassCastException(String.format(
-          "DesiredCapability %s's value should be boolean: found value %s of type %s", key, o.toString(), o.getClass()
-              .getName()));
+      return is(key);
```

Comment:
hmm, that method makes somewhat sense in core java, but I'm not sure I like that behavior for selendroid. We're expecting a boolean value, but essentially we only care about either true or "true" coming across... everything else is false. Is it going to be confusing to any users?

I'd almost rather add an additional else clause to check for the string of "true" or "false", otherwise raise the exception.

Since users of other programming languages (like javascript or python) may have a different impression of what a truthy value should be.